tools/generator/src/Generator.swift

```diff
@@ -0,0 +1,52 @@
+import XcodeProj
+
+class Generator {
+    static let defaultEnvironment = Environment(
+        createProject: Generator.createProject
+    )
+
+    static func createProject(project: Project) -> (PBXProj, PBXProject) {
+        let pbxProj = PBXProj()
+
+        let mainGroup = PBXGroup()
+        pbxProj.add(object: mainGroup)
+
+        let buildConfigurationList = XCConfigurationList()
+        pbxProj.add(object: buildConfigurationList)
+
+        let attributes = [
+            "BuildIndependentTargetsInParallel": 1,
+            // TODO: Generate these
+            "LastSwiftUpdateCheck": 1320,
+            "LastUpgradeCheck": 1320,
+        ]
+
+        let pbxProject = PBXProject(
+            name: project.name,
+            buildConfigurationList: buildConfigurationList,
+            // TODO: Calculate `compatibilityVersion`
+            compatibilityVersion: "Xcode 13.0",
+            mainGroup: mainGroup,
+            // TODO: Make regions configurable?
```

Comment:
All three of these TODOs are staying for now. Not sure if they will be resolved before MVP (low priority).